neptune-core/src/application/util_proof/sent/mod.rs

```diff
@@ -0,0 +1,329 @@
+/* `sent_transactions` method allows to get the info by the pair of numbers of the tx and the UTXO in it */
+
+/*
+- ✔️ output the address lock part
+- ✔️ output the ammount(s)
+- ✔️ check the UTXO canonical commitment is in the AOCL
+- ✔️ output the digest of the AOCL
+- ✔️ output the digest of the `sender_randomness` (as reusing it won't allow to put the UTXO again, and its sole role is shifting the AR)
+- ✔️ the recipient digest is in the public inputs */
+
+use std::sync::OnceLock;
+
+use tasm_lib::{
+    field as rustfield,
+    memory::{encode_to_memory, FIRST_NON_DETERMINISTICALLY_INITIALIZED_MEMORY_ADDRESS},
+    prelude::{Digest, Library, TasmObject, TasmStruct},
+    triton_vm::{
+        isa::triton_asm,
+        prelude::{BFieldCodec, LabelledInstruction, Program},
+        proof::Claim,
+        vm::{NonDeterminism, PublicInput},
+    },
+    twenty_first::util_types::mmr::mmr_accumulator::MmrAccumulator,
+};
```

Comment:
We use "item" granularity for imports to avoid headaches when solving git merge conflicts.